spec/services/google_scraping_service_spec.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'vcr'
+
+RSpec.describe GoogleScrapingService, type: :service do
+  include ActiveJob::TestHelper
+  ActiveJob::Base.queue_adapter = :test
```

Comment:
can we set this on global, maybe Rspec config instead of putting it in each test file

Reply:
Fixed in cd698477f28ffec45a1c3f9fbd4738ad24758f95